.github/workflows/scripts/check_docstrings.py

```
@@ -0,0 +1,773 @@
+    ignores = ["test_", "tearDownClass", "setUpClass", "setUp", "tearDown"]
+
+    # Ignore test functions in test files
+    for ignore in ignores:
+        if function.name.startswith(ignore) and ("test_" in file_path):
+            result = True
+
+    # Return
+    return result
+
+
+def match_arguments_to_docstring(function, docstring, line_number):
```

Ruff (0.12.2)

`match_arguments_to_docstring` is too complex (11 > 10)

(C901)